lnbits/extensions/tpos/tasks.py

```diff
@@ -60,7 +60,7 @@ async def on_invoice_paid(payment: Payment) -> None:
         payment_request="",
         payment_hash=payment.payment_hash,
         amount=tipAmount,
-        memo=payment.memo,
+        memo="Tip",
```

Comment:
May I propose here doing something like `f"Tip {payment.memo}"` here? In my use case, `payment.memo` has a unique identifier for a specific order. This helps the business just reconcile everything.

Reply:
Fixed.